README.md

```diff
@@ -90,10 +90,27 @@ See [i18n](http://ant.design/docs/react/i18n).
 
 ## Development
 
+Clone the repo:
 ```bash
-$ git clone git@github.com:ant-design/ant-design.git
```

Comment:
The missing  command is `cd ant-design`. Don't need to mention `yarn` here, people who using `yarn` must know they can replace `npm` with `yarn`.